examples/06-Multiphysics/Maxwell3D_Icepak_2Way_Coupling.py

```diff
@@ -0,0 +1,195 @@
+)
+
+###############################################################################
+# Set conductivity temperature coefficient
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Conductivity is now function of temperature
+
+Cu = m3d.materials["copper"]
+cu_resistivity_temp_coefficient = 0.00393
+Cu.conductivity.add_thermal_modifier_free_form("1.0/(1.0+{}*(Temp-20))".format(cu_resistivity_temp_coefficient))
+
+###############################################################################
+# Set object temperature and enable feedback
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Fro 2-way coupling
+
+m3d.modeler.set_objects_temperature(["Coil"])
+
+###############################################################################
+# Assign matrix
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# RL calculation
+
+m3d.assign_matrix(["Winding1"], matrix_name="Matrix1")
+
+###############################################################################
+# Create and analyze simulation setup
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Simulation frequency 150kHz
+
+setup = m3d.create_setup(setupname="Setup1")
+setup.props["Frequency"] = "150kHz"
+m3d.analyze_setup("Setup1")
+
+# todo: how to get the resistance?
+#resistance = m3d.post.get_solution_data("Matrix1.R(Winding1,Winding1)").full_matrix_mag_phase[0]
+#m3d.logger.design_logger.info("Coil resistance without temperature feedback =  %f", resistance)
+
+###############################################################################
+# Icepak design
+# ~~~~~~~~~~~~~
+# Insert icepak design and copy solid objects from Maxwell, modify region dimensions
+
+# desktop = m3d.desktop_class
+# ipk = desktop[[m3d.project_name, icepak_design_name]]
+
+ipk = pyaedt.Icepak(designname=icepak_design_name)
+ipk.copy_solid_bodies_from(m3d, no_pec=False)
+ipk.modeler.edit_region_dimensions([50, 50, 50, 50, 500, 500])
+
+###############################################################################
+# Map coil losses
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Map ohmic losses in coil
+
+ipk.assign_em_losses(
+    designname="1 Maxwell",
+    setupname=m3d.setups[0].name,
+    sweepname="LastAdaptive",
+    object_list=["Coil"],
+)
+
+###############################################################################
+# Boundary conditions
+# ~~~~~~~~~~~~~~~~~~~
+# Assign opening
+
+faces = ipk.modeler["Region"].faces
+face_names = [face.id for face in faces]
+ipk.assign_free_opening(face_names, boundary_name="Opening1")
+
+###############################################################################
+# Icepak mesh
+# ~~~~~~~~~~~~~~~~~~~
+# Assign mesh
+
+mesh_region = ipk.mesh.assign_mesh_region(objectlist=["Coil", "Core"])
+mesh_region.UserSpecifiedSettings = True
+mesh_region.MinGapX = "1mm"
+mesh_region.MinGapY = "1mm"
+mesh_region.MinGapZ = "1mm"
+mesh_region.MaxLevels = "2"
+mesh_region.MaxElementSizeX = "10mm"
+mesh_region.MaxElementSizeY = "10mm"
+mesh_region.MaxElementSizeZ = "10mm"
+mesh_region.update()
+
+
```

Comment:
```suggestion
```

I checked the automatic mesh that Icepak creates. It is good enough, so we can remove these lines.